nservicebus/dotnet-templates/index_sc-transport-adapter_templates_[,5).partial.md

```diff
@@ -0,0 +1,22 @@
+## ServiceControl Transport Adapter Windows Service
```

Comment:
With the bridge we no longer recommend SC transport adapters to be used, remove?

Reply:
makes sense 👍